eox_nelp/api_clients/tests/tests_futurex.py

```diff
@@ -27,7 +28,7 @@ def test_enrollment_progress(self, auth_mock, post_mock):
             - Response is the expected value
             - make_post was called with the right values.
         """
-        auth_mock.return_value = {}
+        auth_mock.return_value = requests.Session()
```

Comment:
Why the use of extra_headers_key. Needs the change of the empty dict to request.Session()